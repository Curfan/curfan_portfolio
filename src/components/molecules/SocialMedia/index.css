.social-media {
	display: flex;
	flex-wrap: wrap;

	& > .link {
		&:not(:last-child) {
			margin-right: 35px;
		}
	}
}
