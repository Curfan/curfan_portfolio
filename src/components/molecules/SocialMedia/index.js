import React from 'react';
import classnames from 'classnames';

import SocialLink from 'components/atoms/SocialLink';

import FacebookIcon from 'images/icon/social-facebook.inline.svg';
import YoutubeIcon from 'images/icon/social-youtube.inline.svg';
import LinkedinIcon from 'images/icon/social-linkedin.inline.svg';

import styles from './index.css';

const SocialMedia = ({ className }) => (
	<div className={classnames(styles.socialMedia, className)}>
		<SocialLink className={styles.link} Icon={FacebookIcon} link="https://www.facebook.com/" />
		<SocialLink className={styles.link} Icon={YoutubeIcon} link="https://www.youtube.com/" />
		<SocialLink className={styles.link} Icon={LinkedinIcon} link="https://www.linkedin.com/" />
	</div>
);

export default SocialMedia;
