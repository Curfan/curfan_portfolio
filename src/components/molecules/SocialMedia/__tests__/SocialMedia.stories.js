import React from 'react';
import { storiesOf } from '@storybook/react';

import SocialMedia from 'components/molecules/SocialMedia';

const stories = storiesOf('molecules/SocialMedia', module);

stories.add('__interactive', () => <SocialMedia />);
