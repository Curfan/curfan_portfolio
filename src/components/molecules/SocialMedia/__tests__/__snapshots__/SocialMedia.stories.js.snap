// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`__interactive 1`] = `
<DocumentFragment>
  <div
    class="socialMedia"
  >
    <a
      class="socialLink link"
      href="https://www.facebook.com/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <test-file-stub />
    </a>
    <a
      class="socialLink link"
      href="https://www.youtube.com/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <test-file-stub />
    </a>
    <a
      class="socialLink link"
      href="https://www.linkedin.com/"
      rel="noopener noreferrer"
      target="_blank"
    >
      <test-file-stub />
    </a>
  </div>
</DocumentFragment>
`;
